outputs/client.go

```diff
@@ -178,6 +178,12 @@ func (c *Client) Post(payload interface{}) error {
 		req.Header.Add("event-namespace", c.Config.Kubeless.Namespace)
 	}
 
+	if c.OutputType == "GCPCloudRun" {
+		if c.Config.GCP.CloudRun.JWT != "" {
```

Comment:
```suggestion
	if c.OutputType == "GCPCloudRun" && c.Config.GCP.CloudRun.JWT != "" {
```

Nested `if` is useless

Reply:
thank you 👍